youtube_dl/extractor/peertube.py

```diff
@@ -528,12 +542,14 @@ def account_data(field):
         return {
             'id': video_id,
             'title': title,
-            'description': video.get('description'),
+            'description': video_description.get('description'),
             'thumbnail': urljoin(url, video.get('thumbnailPath')),
             'timestamp': unified_timestamp(video.get('publishedAt')),
             'uploader': account_data('displayName'),
-            'uploader_id': account_data('uuid'),
-            'uploder_url': account_data('url'),
+            'uploader_id': '%s@%s' % (account_data('name'), account_data('host')),
```

Comment:
That's not a id.

Reply:
There appears to be no in-built globally-unique id that is consistent across PeerTube instances.
If this is not an issue, I can use the int 'id' instead.

Reply:
It's not supposed to be globally unique. Read meta field description.

Reply:
Each PeerTube server has a domain and locally ensures user uniqueness, since instances do not coordinate for id generation. Hence, much like with email, the `name@host` is the closest we will get to an id.

@3risian note the `uploader_id` should be the channel's, not the account's.

Reply:
Noted.
The current implementation uses the account name and account URL instead of channel name and URL. Should I change those?